### PR TITLE
docs(S3): update faq & billing MTA-3847

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: faq/new-object-storage-classes.mdx [hunk 1]
```diff
@@ -29,23 +34,15 @@ New objects created after the Multi-AZ launch can have the `Standard (Multi-AZ)`
...
 No, any objects you upload into your buckets are added as Standard class objects by default, just as before. If you want to transfer your objects to One Zone - IA or Glacier you can do so manually or set up [lifecycle rules](/storage/object/api-cli/lifecycle-rules-api/). 
 
 </Concept>
 
 <Concept>
 
-## I do not want to use the new Standard class but I have a bucket and objects in PAR or AMS. Can I still use the old class?
-Unfortunately, you cannot use the old Standard class. You must use the [One Zone - IA class](/storage/object/concepts/#storage-class).
+## I do not want to use the new Standard (Multi-AZ) class. Can I still use the old class?
+Unfortunately, you cannot use the old Standard class. You must use the [One Zone - IA class](/storage/object/concepts/#storage-class), which has the same behavior.
```
Comment (bene2k1) [inline faq/new-object-storage-classes.mdx:45]:
```suggestion
Unfortunately, you cannot use the old Standard class. You must use the [One Zone - IA class](/storage/object/concepts/#storage-class), which has the same behavior as the old Standard class.
```

[the remark this answers is not in the document]
Comment (SamyOubouaziz) [reply faq/new-object-storage-classes.mdx:45]: Old standard did not have multi AZ replication. Old standard = new OneZoneIA !

Comment (RoRoJ) [reply faq/new-object-storage-classes.mdx:45]: Ahh I read too fast before ;-)

[the remark this answers is not in the document]
Comment (RoRoJ) [reply faq/new-object-storage-classes.mdx]: I read too fast before, sorry!

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**